Modelica_DeviceDrivers/Blocks/Communication.mo

```diff
@@ -270,24 +271,24 @@ See <a href=\"modelica://Modelica_DeviceDrivers.Blocks.Examples.TestSerialPackag
     parameter Boolean autoBufferSize = true
       "true, buffer size is deduced automatically, otherwise set it manually."
       annotation(Dialog(group="Outgoing data"), choices(checkBox=true));
-    parameter Integer userBufferSize=16*64
-      "Buffer size of message data in bytes (if not deduced automatically)." annotation(Dialog(enable=not autoBufferSize, group="Outgoing data"));
+    parameter Integer userBufferSize=16*64 "Buffer size of message data in bytes (if not deduced automatically)."
+      annotation(Dialog(enable=not autoBufferSize, group="Outgoing data"));
     parameter String Serial_Port="/dev/ttyPS0" "SerialPort to sendData"
       annotation (Dialog(group="Outgoing data"));
-    parameter SerialBaudRate baud = SerialBaudRate.B9600
-      "Serial port baud rate"
-       annotation (Dialog(group="Outgoing data"));
-    parameter Integer parity = 0
-      "set parity (0 - no parity, 1 - even, 2 - odd)"
-       annotation (Dialog(group="Outgoing data"));
+    parameter SerialBaudRate baud = SerialBaudRate.B9600 "Serial port baud rate"
+      annotation (Dialog(group="Outgoing data"));
+    parameter Integer parity = 0 "set parity (0 - no parity, 1 - even, 2 - odd)"
+      annotation (Dialog(group="Outgoing data"));
+    parameter Integer byteSize(min=5, max=8) = 8 "Byte size"
```

Comment:
Maybe add a bit more explanation. Although the term `byteSize` is what is used in serial port programming, it just feels strange to specify that a byte has a different number of bits than 8. Maybe following helps a "non-expert" user:
"Number of data bits transmitted per (serial data format) byte  (8 is most common)"